### DISCUSSION TITLE
Qgroundcontrol vidéo streaming

### DISCUSSION THREAD

**Discussion by maguet11020** [bluenviron/mediamtx#3970] [Questions]
### Question

Hi,
I was able
To get working my px4 fc with ( thanks to) mediamtx with a raspberry pi . I get the video streaming but I’m unable
To get it on qgroundcontrol . Is there a solution ? Thanks

**Comment by aler9** [comment]
The question is not clear. If you still need help, open a question and provide software version and clear instructions on how to replicate the issue.
